ui/src/app/workflows/components/submit-workflow-panel.tsx

```diff
@@ -62,16 +63,17 @@ export function SubmitWorkflowPanel(props: Props) {
         }
     }
 
+    const {t} = useTranslation();
```

Comment:
same as above, this should be with the variable declarations at the top of the component